ci/jenkins/jobs/projects.yaml

```diff
@@ -1147,3 +1147,59 @@
              - timeout:
                  timeout: 600
                  type: absolute
+      - '{name}-{test_name}-for-pull-request':
```

Comment:
Does VMBM need to support networkpolicy and conformance test on new testbed? @wenyingd 
If so I think we need to add them as well.

Reply:
Later we will be adding networkpolicy test in the VMAgentTest as well, but I think it will be under the same test name. Not sure about conformance

Reply:
No, I don't think K8s conformance/networkpolicy test are suitable for VM cases. We don't need them